src/collection-view.js

```diff
@@ -544,10 +544,15 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // Destroy the child views that this collection view
   // is holding on to, if any
-  destroyChildren: function() {
+  destroyChildren: function(checkEmpty) {
```

Comment:
Yeah good spot, I'll sort that

On Saturday, 28 February 2015, Christian Maher <notifications@github.com>
wrote:

> In src/collection-view.js
> <https://github.com/marionettejs/backbone.marionette/pull/2384#discussion_r25554274>
> :
>
> > @@ -544,10 +544,15 @@ Marionette.CollectionView = Marionette.View.extend({
> >
> >    // Destroy the child views that this collection view
> >    // is holding on to, if any
> > -  destroyChildren: function() {
> > +  destroyChildren: function(checkEmpty) {
>
> since this is a public api, we should probably accept options instead of
> just a boolean field.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marionettejs/backbone.marionette/pull/2384/files#r25554274>
> .
>